specification/kubernetesconfiguration/resource-manager/readme.md

```diff
@@ -28,9 +28,17 @@ These are the global settings for the KubernetesConfiguration.
 title: SourceControlConfigurationClient
 description: KubernetesConfiguration Client
 openapi-type: arm
-tag: package-2021-03
+tag: package-preview-2020-07-extensions
 ```
 
+### Tag: package-preview-2020-07-extensions
+
+These settings apply only when `--tag=package-preview-2020-07-extensions` is specified on the command line.
+
+```yaml $(tag) == 'package-preview-2020-07-extensions'
+input-file:
+  - Microsoft.KubernetesConfiguration/preview/2020-07-01-preview/extensions.json
```

Comment:
If you only set `Microsoft.KubernetesConfiguration/preview/2020-07-01-preview/extensions.json` as input file, once we release a new SDK, it will only contain features in `Microsoft.KubernetesConfiguration/preview/2020-07-01-preview/extensions.json`, but no features in `kubernetesconfiguration.json`, is it what you expected?
From users' perspective, if they use the new SDK, they will be broken as features in `kubernetesconfiguration.json` are missing in new SDK.